src/scratchtocatrobat/converter/converter.py

```diff
@@ -1529,7 +1522,8 @@ def _convert_append_to_list_block(self):
     def _convert_insert_at_of_list_block(self):
         [value, position, list_name] = self.arguments
         if position == "last":
-            index_formula = catrobat.create_formula_with_value(self._converted_helper_brick_or_formula_element([list_name], "lineCountOfList:"))
+            index_fe_linecount = self._converted_helper_brick_or_formula_element([list_name], "lineCountOfList:")
+            index_formula = catrobat.create_formula_with_value(self._converted_helper_brick_or_formula_element([index_fe_linecount, 1], "+"))
```

Comment:
I'm wondering why it should be `n+1` instead of `n` since this seems to be out of range unless the first index always starts with 2 instead of 1.
Therefore, I'd recommend to replace these two lines with the following line:
`return self._converted_helper_brick_or_formula_element([value, list_name], "append:toList:")`
as suggested by Wolfgang.